dfastmi/gui.py

```diff
@@ -332,13 +333,15 @@ def updated_reach(ireach: int) -> None:
     """
     ibranch = dialog["branch"].currentIndex()
 
-    q_stagnant = rivers.qstagnant[ibranch][ireach]
+    reach = rivers.branches[ibranch].reaches[ireach]
```

Comment:
e.g. similar as above but then for reach index instead of reach.

Reply:
I agree, but gui.py is not in scope of this issue. I only refactor code to get it working again. Will pickup the good comments later in new issue

Reply:
Discussed, we will leave these comments for a later refactoring.
When a new issue is created for this, the number will be disclosed here.